src/iuse.cpp

```diff
@@ -4612,7 +4591,7 @@ int iuse::portable_game(player *p, item *it, bool, const tripoint& )
     if (p->has_trait("ILLITERATE")) {
         add_msg(_("You're illiterate!"));
         return 0;
-    } else if (it->charges < 15) {
+    } else if (it->ammo_remaining() < 15) {
```

Comment:
It takes a while to use, so I guess using it at 5 battery would be a waste of time.